pkg/cmd/completion/completion.sh

```diff
@@ -9,6 +9,23 @@ __dcos_debug() {
     fi
 }
 
+
+# checks if an array of subcommands contains a subcommand.
+# usage: _contains_subcommand <arrays_of_subcommands> <cmd>
+_contains_subcommand () {
+    local possible_cmds="$1[@]"
```

Comment:
I'm not sure this works. It looks like you're trying to use https://wiki.bash-hackers.org/syntax/arrays#indirection which isn't a bad idea but that's not the syntax to accomplish it. Though the syntax given there makes shellcheck real unhappy.

```
asdf() {
    local foo="$1[@]"
    echo "${foo[@]}"
}

bar=("a" "b" "c" "d")
asdf bar
```
Gives
```
❯ bash test.sh
bar[@]
```

Reply:
Yeah something is not working yet. I'll fix that 🤦‍♂️